src/PKSim.Presentation/Presenters/Compounds/CompoundInSimulationPresenter.cs

```diff
@@ -26,6 +26,8 @@ public class CompoundInSimulationPresenter : AbstractCommandCollectorPresenter<I
       private List<IParameter> _allParameters;
       public string Description { get; set; }
       public bool ForcesDisplay => true;
+      public bool AlwaysRefresh { get; } = false;
```

Comment:
This flag is now in the interface. False for all except for our list of changed parameters that refreshes all the time. It is an optimization that we need to bypass in this case